README.md

```diff
@@ -45,6 +45,7 @@ an existing package to this repository.
 
 - [Clock](/clock) provides an interface for time-based operations.  It allows
   mocking time for testing.
+- [Pointers](/pointers) provides some functions for pointer-based operations.
```

Comment:
For some reason, we've kept an empty line between each module (don't ask me why), but maybe we should keep it consistent?